.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+
+python:
+  - "3.5"
+
+install:
+  - pip install bubuku
```

Comment:
And as far as I understand - install section is needed to install dependencies, not project by itself.